appveyor.yml

```diff
@@ -2,70 +2,75 @@
-#      VS_VERSION: "9 2008"
-#      arch: Win64
+    # Python 2.7
+    # - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    #   VS_VERSION: "9 2008"
+    #   ARCH: Win32
+    # - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    #   VS_VERSION: "9 2008"
+    #   ARCH: Win64
     # Python 3.4
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       VS_VERSION: "10 2010"
+      ARCH: Win32
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      VS_VERSION: "10 2010"
+      ARCH: Win64
+    # Not used as a python compiler
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      VS_VERSION: "12 2013"
       arch: Win32
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-#      VS_VERSION: "10 2010"
-#      arch: Win64
-#    # Not used as a python compiler
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-#      VS_VERSION: "12 2013"
-#      arch: Win32
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-#      VS_VERSION: "12 2013"
-#      arch: Win64
-#    # Python 3.5 and 3.6
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-#      VS_VERSION: "14 2015"
-#      arch: Win32
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-#      VS_VERSION: "14 2015"
-#      arch: Win64
-#    # Not used as a python compiler
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-#      VS_VERSION: "15 2017"
-#      arch: Win32
-#    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-#      VS_VERSION: "15 2017"
-#      arch: Win64
-  
-matrix:
-  fast_finish: false
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      VS_VERSION: "12 2013"
+      ARCH: Win64
+    # Python 3.5 and 3.6
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      VS_VERSION: "14 2015"
+      ARCH: Win32
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      VS_VERSION: "14 2015"
+      ARCH: Win64
+    # Not used as a python compiler. Swig is not working with these compilers
+    # - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    #   VS_VERSION: "15 2017"
+    #   ARCH: Win32
+    # - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    #   VS_VERSION: "15 2017"
+    #   ARCH: Win64
 
-# skip unsupported combinations
 init:
-  - set _arch=
-  - if "%arch%"=="Win64" ( set _arch= Win64)
-  - echo %_arch%
+  - set SWMM_HOME=C:\projects\stormwater-management-model\
+  - set BUILD_HOME=build
+  - if "%ARCH%"=="Win64" ( set VS_ARCH= Win64)
+  - set GENERATOR="Visual Studio %VS_VERSION%%VS_ARCH%"
+  # See set values
   - echo %APPVEYOR_BUILD_WORKER_IMAGE%
-  - set generator="Visual Studio %VS_VERSION%%_arch%" 
-  - echo %generator%
+  - echo %ARCH%
+  - echo %GENERATOR%
+
+install:
+  # Install SWIG using chocolatey
+  - choco install swig
+  - swig -version
+  # `C:\Python27` (x86) is default on path for `python` command
+  - python --version
+  # Install nrtest and dependencies for regression tests
+  - python -m pip install --src build\packages -r tools\requirements.txt
```

Comment:
@michaeltryby is there a reason why the requirements point to a repo fork of the nrtest repo (Inside OpenWaterAnalytics) instead of the original repo. I ask because

- I saw (on quick insection) no difference between the fork and the original repo 
- It seems the pypi nrtest package could be used directly?

Reply:
@goanpeca I reported a bug to `nrtest` related to command line parsing of the `rtol` and `atol` values. They incorporated the patch but didn't update the version on pypi. Because there were other changes we wanted to make to `nrtest` anyway @bemcdonnell and I decided to fork it.  

Are you trying to optimize the speed of the build?

Reply:
Nope, it was just a question I had, now that is clear I will add a comment to the requirements file 
:-)